src/Oscar.jl

```diff
@@ -28,6 +28,32 @@ const jll_deps = String["Antic_jll", "Arb_jll", "Calcium_jll", "FLINT_jll", "GAP
                         "libpolymake_julia_jll", "libsingular_julia_jll",
                         "polymake_jll", "Singular_jll"];
 
+# We read experimental and filter out all packages that follow our desired
+# scheme. Remember those packages to avoid doing this all over again for docs
+# and test.
+# We don't want to interfere with existing stuff in experimental though.
+const expdir = joinpath(@__DIR__, "../experimental")
+const oldexppkgs = [
+  "ExteriorAlgebra", "GaloisGrp", "GITFans", "GModule", "JuLie",
+  "LinearQuotients", "Matrix", "ModStd", "MPolyRingSparse",
+  "Rings", "Schemes", "SymmetricIntersections", "FTheoryTools"
```

Comment:
Super unimportant nitpick suggestion:

maybe put one per line so that whenever we convert one to the new style, we just have to delete a line. Also sort alphabetically?
```suggestion
  "ExteriorAlgebra",
  "FTheoryTools",
  "GaloisGrp",
  "GITFans",
  "GModule",
  "JuLie",
  "LinearQuotients",
  "Matrix",
  "ModStd",
  "MPolyRingSparse",
  "Rings",
  "Schemes",
  "SymmetricIntersections",
```